scripts/buildProjects.ps1

```diff
@@ -32,6 +32,10 @@ function BuildServer ( $srcDir, $outDir, $target) {
         $commandArgs += "/p:UseAppHost=false"
     }
 
+    if ($env:ALLOW_ENCRYPTED_SSL_ONLY) {
```

Comment:
Would prefer that to be just a switch in the build script

Reply:
Done